contributing.md

```diff
@@ -71,10 +71,16 @@ This document provides some overall guidelines and suggestions for how to get st
 
 * Check that your local branch of development is up-to-date before making the pull request. You can do this while having your feature branch checked out and by using `git pull --rebase origin development`. We recommend a rebase strategy to keep at a minimum the number of commits which need to be reviewed for a given PR. A full explanation can be explored [here](https://www.atlassian.com/git/tutorials/merging-vs-rebasing)
   * The rebase operation may cause merge conflicts. **This is a good thing.** When you proactively resolve merge conflicts on your featuer branch, it allows our hard-working maintainers to cleanly merge your PR into development. 
-  * You will need to resolve and commit merge conflicts locally and ensure your modified code continues to function as expected. And remember ... you are the most qualified person to handle merge conflicts for the code you've written!
-* Now, push your changes to the repository on Github: `git push`  
+  * You will need to resolve and commit merge conflicts locally and ensure your modified code continues to function as expected. You can use `git add .` then `git commit -m "description"`.  And remember ... you are the most qualified person to handle merge conflicts for the code you've written!
+* Return to your branch with `git rebase --continue`
+* Check the commit log to make sure you're only pushing one commit for the branch you're working on using `git log`.
```

Comment:
I changed this line to "Verify that the your desired changes were successful using `git log`" since below it were sub-bullets, and updated those as well~